pkg/skaffold/deploy/status_check.go

```diff
@@ -171,6 +180,38 @@ func printStatusCheckSummary(d *resource.Deployment, pending int, total int, err
 	color.Default.Fprintln(out, status)
 }
 
+// Print resource statuses until all status check are completed or context is cancelled.
+func printResourceStatus(ctx context.Context, deps []*resource.Deployment, deadline time.Duration, out io.Writer) {
```

Comment:
nit: usually we do the context, then the writer, then the other stuff.

`func printResourceStatus(ctx context.Context, out io.Writer, deps []*resource.Deployment, deadline time.Duration)`

Reply:
sounds good.